fixture-monkey/src/main/java/com/navercorp/fixturemonkey/arbitrary/ArbitraryTree.java

```diff
@@ -63,6 +64,9 @@ public ArbitraryTree(ArbitraryNode<T> head) {
 			selectNodes.addAll(nextNodes);
 			nextNodes.clear();
 		}
+		List selectNodesList = new ArrayList(selectNodes);
+		Collections.shuffle(selectNodesList);
+		selectNodes = new LinkedList<>(selectNodesList);
```

Comment:
I think it is better to switch `selectNodes` as `LinkedList` from `Queue`.
Because `selectNodes` should be an implementation of `Queue` and `List` in this case.

It is better to use `Random` instance looked up by `Randoms` for reproducing same results when seed is fixed